tests/nemo_text_processing/de/test_sparrowhawk_inverse_text_normalization.sh

```diff
@@ -3,7 +3,7 @@
 PROJECT_DIR=/workspace/tests
 
 GRAMMARS_DIR=${1:-"/workspace/sparrowhawk/documentation/grammars"}
-PROJECT_DIR=${2:-"/workspace/tests/en"}
+PROJECT_DIR=${2:-"/workspace/tests/"}
```

Comment:
Why is this necessary?

Reply:
Sparrowhawk throws an error otherwise.  I described it [here](https://docs.google.com/document/d/1IWm_rLK0dHe0E2aewdAf63EW3doV47JLV52e_yRsyLs/edit).

Reply:
@zoobereq could you please add DE tests to SH? I don't see them in Jenkins
Have you tried `/workspace/tests/de`?

Reply:
@ekmb they are referenced in `de/test_sparrowhawk_normalization.sh` and `de/test_sparrowhawk_inverse_text_normalization.sh`.  They are also referenced in `JenkinsFile` -- what am I missing?
And yes, I did try `/workspace/tests/de` and received:
`test_sparrowhawk_normalization.sh: line 86: /workspace/tests/en/../shunit2/shunit2: No such file or directory
` (also described [here](https://docs.google.com/document/d/1IWm_rLK0dHe0E2aewdAf63EW3doV47JLV52e_yRsyLs/edit))